WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -77,6 +79,18 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
     var checkoutButtonDisabled: Bool {
         return itemsInCart.isEmpty
     }
+
+    func cardPaymentTapped() {
+        Task { @MainActor in
+            showsCreatingOrderSheet = true
+            let order = try await createTestOrder()
+            showsCreatingOrderSheet = false
+            let _ = try await cardPresentPaymentService.collectPayment(for: order, using: .bluetooth)
+
+            // TODO: Here we should present something to show the payment was successful or not,
+            // and then clear the screen ready for the next transaction.
```

Comment:
For reference, this is the design to show after a successful payment.

TfaZ4LUkEwEGrxfnEFzvJj/WooCommerce-POS?node-id=653-32958&t=MvK9jHYAj1UXyCVM-0

<img width="341" alt="Screenshot 2024-06-05 at 1 34 54 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/2b9f55bd-55d6-4af0-b7db-344538d578fb">